ucsd-fair-data-informatics.md

```diff
@@ -0,0 +1,67 @@
+# UCSD Fair Data Informatics Lab - Neuroscience Information Framework
+## Description
+The University of California, San Diego (UCSD) Fair Data and Informatics Laboratory (FDI Lab) supports open science and is 
+home to the Neuroscience Information Framework (NIF). NIF has extensive experience in community
+engagement, collaborative standards building, effective processes for sharing, federating, and
+integrating neuroscience data, information, and knowledge.
+
+# Roles
+[Principal Investigator] Maryann Martone (maryann@ncmir.ucsd.edu)   
+[Co-Investigator] Jeffrey Grethe (jgrethe@ncmir.ucsd.edu)   
+[Co-Investigator] Anita Bandrowski (abandrowski@ncmir.ucsd.edu)  
+[Knowledge Engineer] Tom Gillespie (tom.h.gillespie@gmail.com)   
+
+## Definitions [Optional]
+* __NIFSTD__ Neuroscience Information Framework (NIF) Standard ontology (NIFSTD) is a neuroscience
+  ontology that maintains an extensive set of terms and concepts important for the domains of
+  neuroscience and biology.
+* __InterLex__ A dynamic lexicon of biomedical terms built on the foundation of NeuroLex
+
+## Objectives
+* Develop a semantic framework for annotating and organizing cell level data by maturing community
+  efforts to create a phenotype-based nervous system cell type ontology and phenotype representation
+* Work with the community to develop and implement spatial and semantic standards for managing
```

Comment:
I might also add something about working with BCDC to adapt ontologies as needed to support BCDC data search and visualization portals.

Reply:
Work with the BCDC to develop an information architecture to meet the needs of the BICCN consortium as a whole and that interoperates with the larger ontology ecosystem.

Reply:
resolved by 8edb24f